src/facade/dragonfly_connection.cc

```diff
@@ -1114,7 +1117,7 @@ void Connection::HandleMigrateRequest() {
     this->Migrate(dest);
   }
 
-  DCHECK(dispatch_q_.empty());
+  // DCHECK(dispatch_q_.empty());
```

Comment:
I have triggered this and I suspect it's not needed, since by the time Migrate returns `dispatch_q_` might be active

Reply:
@dranikpg  to validate

Reply:
Maybe uncomment this as it's not related